### DISCUSSION TITLE
Snap all windows of an application

### DISCUSSION THREAD

**Discussion by mfbx9da4** [rxhanson/Rectangle#1341] [Ideas]
I often have to cycle through windows of an application to move them because command tab won't work if I only move one

**Comment by rxhanson** [comment]
Thanks for the feedback. Rectangle is intentionally slim, and I don't have plans to add that to it at this time.

[Rectangle Pro](https://rectangleapp.com/pro) (paid) is where I've added in multiple-window actions as well as a way to save off a layout of windows and re-apply them.
